package.json

```diff
@@ -78,7 +78,8 @@
   },
   "dependencies": {
     "lodash": "^3.10.1",
-    "lodash-compat": "^3.10.1"
+    "lodash-compat": "^3.10.1",
+    "qs": "5.1.0"
```

Comment:
We can (and should) now use qs: ^5.1.0. I know this is the opposite of what we decided some months ago but this is in the end the best way to avoid duplicate dependencies for front end developers when they do build with browserify and webpack..

We now have the shinkwrap to handle our build the right way (reproducible).